heareval/tasks/pipeline.py

```diff
@@ -563,23 +563,18 @@ def run(self):
         self.mark_complete()
 
 
-class SplitData(MetadataTask):
+class SubcorpusData(MetadataTask):
```

Comment:
This name was really confusing. This task doesn't "split" anything.

Reply:
It does separate out all the data (audio files) into folders based on their splits. So in a sense it is splitting the audio files. But I guess this represents the final data for the subsampled corpus.

Reply:
Split means a particular thing in our terminology. It's a bad name because we were using "Split" as an adjective, not a verb